### PR TITLE
docs: update internal absolute links

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?
- Documentation fixes

## What is the current behavior?
Using absolute links makes it hard to test the documentation when using the staging builds. Also, using Markdown document linking ensures the documents are at the right location.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by agneszitte** [comment]
@jeromelaban I will let you review this PR as well

File: doc/articles/get-started-next-steps.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Microsoft MVP Skye Hoefling put together [Uno Platform: Getting Started Series](
 
 ### Raspberry Pi Applications with Uno Platform (Free)
 
-If you are looking to bring your cross-platform apps to Raspberry Pi with Uno Platform, Peter Gallagher wrote an [article to help you get started]( https://www.petecodes.co.uk/developing-uwp-apps-for-the-raspberry-pi-with-uno-platform/) or try our [getting started with Raspberry Pi documentation]( https://platform.uno/docs/articles/guides/raspberry-pi/raspberry-pi-intro.html).
+If you are looking to bring your cross-platform apps to Raspberry Pi with Uno Platform, Peter Gallagher wrote an [article to help you get started]( https://www.petecodes.co.uk/developing-uwp-apps-for-the-raspberry-pi-with-uno-platform/) or try our [getting started with Raspberry Pi documentation](/articles/guides/raspberry-pi/raspberry-pi-intro.md).
```
Comment (Youssef1313) [inline doc/articles/get-started-next-steps.md:68]: The link doesn't work on staging. Maybe:

```suggestion
If you are looking to bring your cross-platform apps to Raspberry Pi with Uno Platform, Peter Gallagher wrote an [article to help you get started]( https://www.petecodes.co.uk/developing-uwp-apps-for-the-raspberry-pi-with-uno-platform/) or try our [getting started with Raspberry Pi documentation](./guides/raspberry-pi/raspberry-pi-intro.md).
```

This comment will apply to multiple changes in the PR

Comment (jeromelaban) [reply doc/articles/get-started-next-steps.md:68]: @ajpinedam it's also best to use `xref:` links instead, wherever possible.

**Comment by morning4coffe-dev** [comment]
Closing in favor of #17705

**Pull request closed without merging**